c2rust-transpile/src/lib.rs

```diff
@@ -182,8 +182,8 @@ impl ExternCrateDetails {
 impl From<ExternCrate> for ExternCrateDetails {
     fn from(extern_crate: ExternCrate) -> Self {
         match extern_crate {
-            ExternCrate::C2RustBitfields => Self::new("c2rust-bitfields", "0.3", true),
-            ExternCrate::C2RustAsmCasts => Self::new("c2rust-asm-casts", "0.2", true),
+            ExternCrate::C2RustBitfields => Self::new("c2rust-bitfields", "0.21", true),
```

Comment:
Can we get the version of the transpiler here, and make sure it always matches?

Reply:
Oh sure, I think we can do that, too.  I'll add a commit.